test/CMakeLists.txt

```
@@ -144,6 +136,11 @@ function(cub_add_test target_name_var test_name test_src cub_target)
   set(config_meta_target ${config_prefix}.tests)
 
   if (is_catch2_test)
+    set(use_rdc_for_catch2_utils OFF)
+  if (CUB_ENABLE_RDC_TESTS OR CUB_FORCE_RDC)
+    set(use_rdc_for_catch2_utils ON)
+  endif()
```

Looks like else branch:
```suggestion
    if (CUB_ENABLE_RDC_TESTS OR CUB_FORCE_RDC)
      set(use_rdc_for_catch2_utils ON)
    endif()
```